p2p/switch_test.go

```diff
@@ -272,10 +272,10 @@ func TestSwitchReconnectsToPersistentPeer(t *testing.T) {
 	// simulate failure by closing connection
 	peer.CloseConn()
 
-	// TODO: actually detect the disconnection and wait for reconnect
+	// TODO: remove sleep, detect the disconnection, wait for reconnect
```

Comment:
make an issue, link to it in code

Reply:
ok thanks https://github.com/tendermint/tendermint/issues/956